docs/content/sidebars/concepts.js

```diff
@@ -94,6 +94,7 @@ const concepts = [
 			},
 			'concepts/grpc-overview',
 			'concepts/graphql-rpc',
+			'concepts/archival-store',
```

Comment:
This location puts the topic under App Developers. Doesn't the content also pertain to data providers as well? If so, I'd suggest putting just below indexing framework